sklearn/linear_model/_ridge.py

```diff
@@ -637,7 +641,7 @@ def _ridge_regression(
 
     # Some callers of this method might pass alpha as single
     # element array which already has been validated.
-    if alpha is not None and not isinstance(alpha, (np.ndarray, tuple)):
+    if alpha is not None and not isinstance(alpha, (np.ndarray)):
```

Comment:
we can probably remove the paranthesis around `np.ndarray`.
We are sure that we don't have any case where the value in the `tuple` have been validated but not transformed to a NumPy array.